### PR TITLE
#7 poetry

### PR DESCRIPTION
**Pull request by abogushov**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "aioworkers-pg"
+version = "0.2.1-alpha.0"
```
Comment (aamalev) [inline pyproject.toml]: Can I use the version from the commit tag?

Comment (abogushov) [reply pyproject.toml]: Yeah, you can use 

pip install aioworkers-pg==0.2.1-alpha.1
pip install aioworkers-pg==0.2.1a1

Comment (abogushov) [reply pyproject.toml]: I published it locally. So now I try to configure publishing with Github Actions.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# aioworkers-pg
+
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/aioworkers-pg)
+![PyPI](https://img.shields.io/pypi/v/aioworkers-pg)
```
Comment (aamalev) [inline README.md:4]: links broke in markdown

**Pull request merged**